rh15d/run_example/keyword.input

```diff
@@ -29,10 +29,11 @@
 # Cut the atmosphere above a certain TMAX?
   15D_DEPTH_ZCUT = TRUE
   15D_TMAX_CUT = 30000.
-
+  
 # Perform depth optimisation for each column (interpolates height scale)?
   15D_DEPTH_REFINE = FALSE
-
+  15D_SMOOTH_OPTIMIZED_GRADIENTS = 5
```

Comment:
It is fine to add this, but please comment it so it does not change the default `keyword.input`.